pkgs/by-name/ya/yazi-unwrapped/update.sh

```diff
@@ -35,7 +35,11 @@ pushd "$SCRIPT_DIR"
 # Build date
 sed -i 's#env.VERGEN_BUILD_DATE = "[^"]*"#env.VERGEN_BUILD_DATE = "'"${latestBuildDate}"'"#' package.nix
 
+# Cargo.lock
+curl -L -o 'Cargo.lock' "https://raw.githubusercontent.com/sxyazi/yazi/$tagName/Cargo.lock"
+git add 'Cargo.lock'
```

Comment:
If the script is gonna use `git`, we must add it to the `nix-shell` packages above, right?

Reply:
Yeah, but `maintainers/scripts/update.nix` has already used `git`, should we add `git` again?

Reply:
Well, it does work without explicitly adding it, so I guess it's fine.